### PR TITLE
IFIX-242: EAT refactored to Expenditure

### PR DESCRIPTION
**Pull request by rahu-eGov**
EAT refactored with Expenditure

### CODE REVIEW

File: domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131700__expenditure_seed_data [hunk 1]
```diff
@@ -1,12 +1,12 @@
-db.eAT.insertMany([
+db.expenditure.insertMany([
 {
     "_id" : "910f4d23-fc63-48fb-a8fb-f0ad567b788a",
-    "_class" : "org.egov.web.models.EAT",
-    "tenantId" : "HR",
+    "_class" : "org.egov.web.models.Expenditure",
```
Comment (rushang7-eGov) [inline domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131700__expenditure_seed_data:4]: Do we need class attribute? If not mandatory, please remove.

File: domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131800__project_seed_data [hunk 2]
```diff
@@ -4,10 +4,10 @@ db.project.insertMany([
...
     "code": "PR2",
     "name": "Project2",
-    "eatId": "EAT2",
-    "departmentId": "DEP2",
-    "locationId": [
-      "L11","L12"
+    "expenditureId": "910f4d23-fc63-48fb-a8fb-f0ad567b788a",
+    "departmentId": "3d9ef18a-361a-40cf-b142-dd6f998e1ad2",
+    "locationIds": [
+      "bddfe094-9902-4c1e-91dc-1c07c0ce926c","7b8dd41c-c6c1-43f2-9794-ce2e3d648eaa"
```
Comment (rushang7-eGov) [inline domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131800__project_seed_data:10]: Keep this array empty. We don't have corresponding uuids in the location service data.

File: domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131800__project_seed_data [hunk 3]
```diff
@@ -22,10 +22,10 @@ db.project.insertMany([
     "tenantId": "PB",
     "code": "PR3",
     "name": "Project3",
-    "eatId": "EAT3",
-    "departmentId": "DEP3",
-    "locationId": [
-      "L111","L121"
+    "expenditureId": "a4c21fc1-28df-494b-8983-baf3c6b027d7",
+    "departmentId": "3d9ef18a-361a-40cf-b142-dd6f998e1ad3",
```
Comment (rushang7-eGov) [inline domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131800__project_seed_data:26]: I assume these departmentId and expenditureId exist in their corresponding seed data. Can you confirm it?

File: domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131700__expenditure_seed_data (hunk 1 above)
Comment (rahu-eGov) [reply domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131700__expenditure_seed_data:4]: It takes automatically, in order to remove it we have do conversion in all entities. that will take time for whole refactoring thing.

File: domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131800__project_seed_data (hunk 3 above)
Comment (rahu-eGov) [reply domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131800__project_seed_data:26]: yes

File: domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131700__expenditure_seed_data (hunk 1 above)
Comment (rushang7-eGov) [reply domain-services/ifix-master-data-service/src/main/resources/db/seed/V20210811131700__expenditure_seed_data:4]: Ok. Let it be like this for now.

**Pull request merged**